lib/cryptoexchange/helpers/hash_helper.rb

```diff
@@ -0,0 +1,20 @@
+module HashHelper
+  class << self
+    # "Polyfill" of the Hash#dig method from Ruby 2.3
+    #
+    #   Examples
+    #
+    #     > dig({a: {b: 1}}, :a, :b)
+    #     => 2
+    #
+    #     > dig({a: {b: 1}}, :a, :c)
+    #     => nil
+    def dig(obj, *args)
```

Comment:
@dominicwong617 Yeah, I agree. I decided against making it a polyfill because I prefer not touching the global scope, especially my implementation isn't 100% compatible with Hash#dig.